src/pnpm-lock.yaml

```diff
@@ -44,17 +44,17 @@ importers:
       typescript: 4.0.5
   packages/cli:
     dependencies:
-      '@prisma/bar': 0.0.1
-      '@prisma/engines': 2.11.0-10.58369335532e47bdcec77a2f1e7c1fb83a463918
+      '@prisma/bar': 0.0.0
+      '@prisma/engines': 2.11.2-9-a624a6628b9e5e453bb8f8cb0460e233a3a97e62
     devDependencies:
       '@prisma/client': 'link:../client'
       '@prisma/debug': 'link:../debug'
       '@prisma/generator-helper': 'link:../generator-helper'
       '@prisma/get-platform': 'link:../get-platform'
       '@prisma/migrate': 'link:../migrate'
       '@prisma/sdk': 'link:../sdk'
-      '@prisma/studio': 0.310.0
-      '@prisma/studio-server': 0.310.0
```

Comment:
This is a bit weird

Reply:
Yes, I just took one of the lock files.
I think it'll be overwritten anyway.